api/config/v1alpha1/shared_types.go

```diff
@@ -67,6 +81,13 @@ type KubernetesServiceSpec struct {
 	// +optional
 	Annotations map[string]string `json:"annotations,omitempty"`
 
+	// Type determines how the Service is exposed. Defaults to LoadBalancer.
+	// Valid options are ClusterIP and LoadBalancer.
+	// "LoadBalancer" means a service will be exposed via an external load balancer (if the cloud provider supports it).
+	// "ClusterIP" means a service will only be accessible inside the cluster, via the cluster IP.
+	// +optional
+	Type ServiceType `json:"type,omitempty"`
```

Comment:
use `ClusterIP` as default value

// +kubebuilder:default:="ClusterIP
// +kubebuilder:validation:Enum=ClusterIP;LoadBalancer

Reply:
The current version uses `LoadBalancer ` by default, IMO change this is not friendly to users who already use it.

Reply:
@arkodg Any other suggestion for this?